cli/fabric.c

```diff
@@ -36,6 +36,26 @@
 #include <sys/types.h>
 #include <errno.h>
 #include <ctype.h>
+#include <inttypes.h>
+
+#ifdef __linux__
+#define PFMTllx	"llx"
+#define PFMTllX	"llX"
+#define PFMTlld	"lld"
+#define PFMThhx	"hhx"
+#define PFMThhu	"hhu"
+#define PFMThx	"hx"
+#define PFMThhd	"hhd"
+#define PFMTlx	"lx"
+#else
+#define PFMTllx	PRIx64
```

Comment:
Aren't these PRI* macros supported on Linux as well? Can't you just use them directly in the printf calls, and you wouldn't need these PFMT defines?

Reply:
It's easier to create our own defines here. Otherwise we have to change the data definition as well. We use a mixture of `uintXX_t` and `unsigned XXX` definitions in our code. Changing these can be a pain.